src/components/ui/PaymentRequestStatusBadge/PaymentRequestStatusBadge.tsx

```diff
@@ -4,40 +4,60 @@ import partiallyPaidIcon from '../../../assets/images/paymentrequest_status_part
 import { cva } from 'class-variance-authority';
 import { LocalPaymentStatus } from '../../../types/LocalTypes';
 
-const badge = cva('rounded-full py-1 px-2 space-x-2 inline-flex items-center', {
+const badge = cva('rounded-full  space-x-2 inline-flex items-center', {
   variants: {
     intent: {
       paid: ['bg-[#D8F2EA]', 'text-[#004D33]'],
       partial: ['bg-[#F2EED8]', 'text-[#663300]'],
       unpaid: ['bg-[#F1F3F4]', 'text-defaultText'],
+      overpaid: ['bg-[#D8F2EA]', 'text-[#004D33]'],
+    },
+    size: {
+      small: ['text-xs', 'font-normal', 'py-1', 'px-2'],
+      large: ['text-sm', 'font-medium', 'leading-[17px]', 'px-4', 'py-2'],
     },
   },
   defaultVariants: {
     intent: 'unpaid',
+    size: 'small',
+  },
+});
+
+const icon = cva('w-auto', {
+  variants: {
+    size: {
+      small: ['h-2'],
+      large: ['h-3'],
+    },
+  },
+  defaultVariants: {
+    size: 'small',
   },
 });
 
 interface PaymentRequestStatusBadgeProps {
   status: LocalPaymentStatus;
+  size: 'small' | 'large';
```

Comment:
Same about getting the type using the helper: https://cva.style/docs/getting-started#variantprops

Reply:
Done.